Umee-Bots/Lending-Pool-Reentrancy/src/utils.ts

```diff
@@ -0,0 +1,32 @@
+import { Finding, FindingSeverity, FindingType } from "forta-agent";
+import { encodeFunctionSignature } from "forta-agent-tools/lib/utils";
```

Comment:
`forta-agent-tools` library is in `devDependencies` and is used outside of development-related situations.